numpy/lib/_npyio_impl.py

```diff
@@ -501,7 +501,7 @@ def _save_dispatcher(file, arr, allow_pickle=None, fix_imports=None):
 
 
 @array_function_dispatch(_save_dispatcher)
-def save(file, arr, allow_pickle=True, fix_imports=True):
+def save(file, arr, allow_pickle=True, fix_imports=None):
```

Comment:
Ah, good point, I hadn't looked close enough.  This is fine, you could also use `np._NoValue` which we usually use as a singleton for this type of thing.

(but also fine to keep it as is, there is no point in passing `None` and it prints nicer in the help/signature)

Reply:
The `fix_imports` parameter is the second keyword parameter with a default value. Therefore, we can't simply remove the `=True` part. Additionally, re-ordering the arguments would introduce incompatibility. 

If we retain the default value, the current code with `fix_imports=True` will not generate a deprecated warning. In this situation, `np.NoValue` is quite handy.